build.sh

```diff
@@ -108,4 +108,11 @@ cpp-client() {
   popd
 }
 
-"$@"
+if [ $# -eq 0 ]
```

Comment:
Be consistent with the rest of the code, the style would be:

if test $# -eq 0; then
  ...
else

[ is an alias for test, which causes people a lot of confusion because it's nothing like () in C/Java.